fri/src/prover/mod.rs

```diff
@@ -222,12 +213,13 @@ where
         });
     }
 
-    /// Creates a FriRemainder from a vector of `evaluations` representing the remainder and
-    /// commits to the latter by hashing sequentially `evaluations`.
-    fn set_remainder(&mut self, channel: &mut C, evaluations: &mut [E]) {
-        let commitment = <H as ElementHasher>::hash_elements(evaluations);
-        channel.commit_fri_layer(commitment);
-        self.remainder = FriRemainder(evaluations.to_vec());
+    /// Creates remainder polynomial from a vector of `evaluations` representing the remainder
```

Comment:
I'd perhaps reword this to emphasize that we create a poly in coeff form, from its evaluations. But not a big deal